frontend/src/App.tsx

```diff
@@ -1,35 +1,14 @@
-import React, {useEffect, useState} from 'react';
+import React from 'react';
 import './App.css';
-import axios from "axios";
-import {BookModel} from "./BookModel";
+import Books from "./Books";
 
 function App() {
 
-    const [books, setBooks] = useState<BookModel[]>([]);
-
-    const fetchAllBooks = () => {
-        axios.get("/api/books")
-            .then(response => response.data)
-            .catch(error => console.log("GET Error: " + error))
-            .then(data => setBooks(data))
-    }
-
-
-    useEffect(() => {
-        fetchAllBooks()
-    }, [])
-
-
-    const bookList = books.map((book) =>
-        <li>Title {book.title}
-            Author {book.author}
-            ISBN {book.isbn}
-            State {book.bookState}
-        </li>
-    )
     return <>
-        <h1>Books</h1>
-        <ul>{bookList}</ul>
+        <div>
```

Comment:
instead of div, maybe use <header><main><footer> or <section> or something of the more semantic HTML-Tags